src/main/java/com/ioet/bpm/skills/domain/converters/CategoryConverter.java

```diff
@@ -0,0 +1,42 @@
+package com.ioet.bpm.skills.domain.converters;
+
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTypeConverter;
+import com.ioet.bpm.skills.domain.Category;
+
+public class CategoryConverter implements DynamoDBTypeConverter<String, Category> {
+
+    @Override
+    public String convert(Category category) {
+        String categoryAsString = null;
```

Comment:
Do we have tests for this code?

Reply:
No, I tried but I could not find how to do it. Do you have any suggestion?

Reply:
I have no idea what this code does. I think it takes an Object and returns it as String, if that's the case you need to check the returned string has the data inside the Object provided (I'd say) .

Reply:
I will move the logic of this method out to another class I can instantiate because I do not know how to DynamoDBTypeConverter should be injected into unit tests.

Reply:
Done.